sqladmin/application.py

```diff
@@ -103,6 +113,22 @@ class UserAdmin(ModelAdmin, model=User):
 
         self._model_admins.append((model()))
 
+    def register_view(self, view: Type["ModelView"]) -> None:
+        class_view = view()
+        class_view.name_plural = class_view.name
+        class_view.url_path_for = self.app.url_path_for
+        class_view.templates = self.templates
```

Comment:
May the `self.templates` can be also added into `register_model` ?

I'm also working on PR to be able to add custom action in the list page for models, and it can be useful :)

Reply:
Thanks for the suggestion, but I think if it's not required here let's keep it in that separate PR.